extensions/functions_geometry.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "st_point"
+    description: >
+      Return a point with the given x and y coordinate values.
+    impls:
+      - args:
+          - name: x
+            value: fp32
+          - name: y
+            value: fp32
+        return: point
+  -
+    name: "st_makeline"
+    description: >
+      Return a linestring containing the points of point, multipoint, or linestring geometries.
+      Repeated points at the beginning of input linestring are collapsed to a single point.
+      Repeated points in point and multipoint inputs are not collapsed.
+    impls:
+      - args:
+          - name: x
+            value: point
+          - name: y
+            value: point
+        return: linestring
+      - args:
+          - name: x
+            value: multipoint
```

Comment:
Is this the same as a list<point>?

Reply:
yeah, it's referred to as a collection in postgis, but for our purposes I think list would be fine.